Actions/AL-Go-Helper.ps1

```diff
@@ -491,29 +493,30 @@ function AnalyzeRepo {
                 throw "No artifacts found for the artifact setting ($artifact) in $ALGoSettingsFile"
             }
             $version = $artifactUrl.Split('/')[4]
+            $storageAccount = $artifactUrl.Split('/')[2]
         }
     
-        if ($settings.supportedCountries -or $country -ne $settings.country) {
+        if ($settings.additionalCountries -or $country -ne $settings.country) {
             if ($country -ne $settings.country) {
                 OutputWarning -message "artifact definition in $ALGoSettingsFile uses a different country ($country) than the country definition ($($settings.country))"
             }
-            Write-Host "Checking Country and SupportedCountries"
+            Write-Host "Checking Country and additionalCountries"
             # AT is the latest published language - use this to determine available country codes (combined with mapping)
             $ver = [Version]$version
             Write-Host "https://$storageAccount/$artifactType/$version/$country"
             $atArtifactUrl = Get-BCArtifactUrl -storageAccount $storageAccount -type $artifactType -country at -version "$($ver.Major).$($ver.Minor)" -select Latest -sasToken $sasToken
             Write-Host "Latest AT artifacts $atArtifactUrl"
             $latestATversion = $atArtifactUrl.Split('/')[4]
-            $countries = Get-BCArtifactUrl -storageAccount $storageAccount -type $artifactType -version $latestATversion -sasToken $sasToken -select All | ForEach-Object { $_.Split('/')[5] }
+            $countries = Get-BCArtifactUrl -storageAccount $storageAccount -type $artifactType -version $latestATversion -sasToken $sasToken -select All | ForEach-Object { $_.Split('?')[0].Split('/')[5] }
```

Comment:
$_.Split('?')[0].Split('/')[5] 
is there any better way of parsing that doesnt need magic numbers? if not can we move it to a function and give it a name